.buildkite/pipelines/pull_request/base.yml

```diff
@@ -169,7 +169,7 @@ steps:
     agents:
       queue: n2-4-virt
     depends_on: build
-    timeout_in_minutes: 60
```

Comment:
Increasing the timeout here is not a good choice. We want to keep our CI under one hour so we should find a way to increase the parallelism, split those or change the agent queue to be kb-n2-8-spot instead \cc @patrykkopycinski

Reply:
agree, Thank you @mistic

Reply:
We will look into a way of setting parallelism dynamically, based on the number of modified test files. For now I rolled back 60m timeout. Thanks for pointing this out!

Reply:
@patrykkopycinski @szwarckonrad I see the burnout tests repeat 5 times and I'm pretty sure it did it a couple of times at first. I have burnout tests interrupted by a timeout [here](https://buildkite.com/elastic/kibana-pull-request/builds/166520#018b18ad-4f44-451f-af8e-d4026986a198) as well. Maybe we can reduce a number of repeats?